src/ui/pages/RequestAccounts/RequestAccounts.tsx

```diff
@@ -303,17 +338,32 @@ export function RequestAccounts() {
   });
 
   if (isError) {
-    return <p>Some Error</p>;
+    if (error instanceof NoWalletForEcosystemError) {
+      const messages: Record<BlockchainType, string> = {
+        solana:
+          'You do not have Solana wallets to connect. Please close this window and visit Settings –> Manage Wallets to create a Solana wallet',
+        evm: 'You do not have Ethereum wallets to connect. Please close this window and visit Settings –> Manage Wallets to create an Ethereum wallet',
+      };
+      return (
+        <EmptyView2
```

Comment:
EmptyView2 ? :)

Reply:
😎  It exists!

Reply:
From the creators of `EmptyView 1` (probably)